deployments/helm/KubeArmorOperator/templates/NOTES.txt

```diff
@@ -0,0 +1,9 @@
+{{- if not .Values.kubearmorOperator.annotateExisting }}
+⚠️  WARNING: Existing pods will not be annotated. Policy enforcement for already existing pods on Apparmor nodes will not work. 
+   • To check enforcer present on nodes use:
+     ➤ kubectl get nodes -o jsonpath='{range .items[*]}{.metadata.name} {.metadata.labels.kubearmor\.io/enforcer}{"\n"}{end}' 
+   • To annotate existing pods use: 
+     ➤ helm upgrade --install {{ .Values.kubearmorOperator.name }} kubearmor/kubearmor-operator -n kubearmor --create-namespace --set annotateExisting=true 
+{{- end }}
```

Comment:
Out controller will rollout restart individual pods to force admission controller to add annotation or you can choose to rollout restart yourself to make your pods have the apparmor annotation and enforcing